README.md

```diff
@@ -8,22 +8,22 @@ Apache Drill is a distributed MPP query layer that supports SQL and alternative
 
 ## Developers
 
-Please read [Environment.md](docs/dev/Environment.md) for setting up and running Apache Drill. For complete developer documentation see [DevDocs.md](docs/dev/DevDocs.md).
+Please read [Environment.md](docs/dev/Environment.md) for setting up and running Apache Drill. For complete developer documentation see [DevDocs.md](docs/dev/DevDocs.md)
```

Comment:
It could be useful to add the link to this wiki for devs:
https://github.com/paul-rogers/drill/wiki
cc: @paul-rogers